test/jdk/java/lang/invoke/callerSensitive/CallerSensitiveAccess.java

```diff
@@ -406,8 +406,9 @@ static Stream<Method> callerSensitiveMethods(Module module) {
         // transform the resource name to a class name
         // load every class in the exported packages
         // return the caller sensitive methods of the public classes
-        try (ModuleReader reader = mref.open()) {
-            return reader.list()
+        try (ModuleReader reader = mref.open();
+             Stream<String> stream = reader.list()) {
+            return stream
```

Comment:
This change is causing the test to fail, in the `callerSensitiveMethods` DataProvider, because the data provider is expecting an open stream to be returned by `callerSensitiveMethods(Module)` - the stream is now closed.

There are a couple of ways to resolve this, but the most straightforward would be to revert this part of the change, and have the `callerSensitiveMethods` DataProvider close the returned stream. E.g.:

```
  @DataProvider(name = "callerSensitiveMethods")
  static Object[][] callerSensitiveMethods() {
      try (var methodStream = callerSensitiveMethods(Object.class.getModule())) {
          return methodStream
                  .map(m -> new Object[]{m, shortDescription(m)})
                  .toArray(Object[][]::new);
      }
  }
```

Reply:
yes, this would solve it.

Reply:
Done in [4a94c64](https://github.com/openjdk/jdk/pull/9557/commits/4a94c645fe1e37abc694f81fd8e401c5dc367d68)

Reply:
@rjernst it seems that the _revert_ part of the above has been done, but not the "have the callerSensitiveMethods DataProvider close the returned stream".

Reply:
Sorry I misunderstood. Fixed now in [3080378](https://github.com/openjdk/jdk/pull/9557/commits/30803780ca0bd1bcf03b99947ebf8cd4b42e6070).